js/build.js

```diff
@@ -2636,6 +2635,22 @@ Fliplet.Widget.instance('chat', function (data) {
       });
   }
 
+  function getUserEmail() {
+    if (!crossLoginColumnName) {
+      return Promise.reject('Cannot find user email. Please review feature configuration.');
+    }
+
+    return Fliplet.User.getCachedSession().then(function (session) {
```

Comment:
Should we simply read from `Fliplet.Profile.get('email')`? What do you think?

Reply:
Because Studio users have provided the data source column, I think directly accessing session data would be the most accurate way of doing this.

Reply:
See https://github.com/Fliplet/fliplet-studio/issues/5169#issuecomment-545407570

Reply:
@tonytlwu my reply on that thread specifically says:

> I can't think of any reason why we shouldn't fallback to reading the email from the profile.

1. keep reading from `Fliplet.App.Storage.set('fl-chat-auth-email')`
2. read from profile or cached session

Reply:
It seems we don't really want to take any risk, so I'll use the session as a fallback instead.